apireader.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import sys
+import os.path
+import requests
+import json
+from argparse import ArgumentParser
+
+DUMP_NAME = "api_data.dump"
+
+
+def get_api_dict(src_dir):
+    with open(src_dir + "directory.json", 'r') as directory_file:
+        api_dict = json.load(directory_file)
+    return api_dict
```

Comment:
How about loading this directly from github and not depending on having that repository around? I mean it's just one file.

```
def get_api_dict(src_dir):
    uri = "https://raw.githubusercontent.com/freifunk/directory.api.freifunk.net/master/directory.json"
    response = requests.get(uri)
    api_dict = json.loads(response.text)
    return api_dict
```